packages/translation/src/lang/en.ts

```diff
@@ -834,6 +865,13 @@ export default {
           },
         },
       },
+      error: {
+        action: {
+          tryAgain: "Try again",
```

Comment:
Shouldn't this be part of a "common" translation file? 
I fear there might be a lot of similar duplicates. 

(But I believe crowdin gets rid of them)

Reply:
You're right try again will probably be used elsewhere